hugegraph-llm/src/tests/models/llms/test_openai_client.py

```
@@ -0,0 +1,250 @@
+    def test_generate_authentication_error(self, mock_openai_class):
+        """Test generate method with authentication error."""
+        # Setup mock client to raise authentication error
+        mock_client = MagicMock()
+        mock_client.chat.completions.create.side_effect = Exception("Authentication error")
+        mock_openai_class.return_value = mock_client
+
+        # Test the method
+        openai_client = OpenAIClient(model_name="gpt-3.5-turbo")
+        
+        # The method should raise an exception for authentication errors
+        with self.assertRaises(Exception):
```

Ruff (0.11.9)

Do not assert blind exception: `Exception`

(B017)